src/main/java/core/basesyntax/CalculatableArea.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface CalculatableArea {

```

Comment:
Like I've mentioned in previous comment, your Figure is neither FigureArea nor CalculatableArea - but could be for example "AreaCalculator" - please give your classes a meaningful names.